src/Controller/ModulesController.php

```diff
@@ -407,6 +407,34 @@ public function relationData($id, string $relation) : void
         $this->set('_serialize', true);
     }
 
+    /**
+     * Load resources of $type callig api `GET /:type/`
+     * Json response
+     *
+     * @param string|int $id the object identifier.
+     * @param string $type the resource type name.
+     * @return void
+     */
+    public function resourcesJson($id, string $type) : void
+    {
+        $this->request->allowMethod(['get']);
+        $response = null;
```

Comment:
Is Scrutinizer right?

Reply:
yes, we may remove the above line, `$response` assignment is don later